module/applications/actor/base-sheet.mjs

```diff
@@ -38,12 +38,28 @@ export default class ActorSheetEd extends ActorSheet {
     return expandObject( enrichment );
   }
 
+  async _enableHTMLEnrichmentItem( itemC ) {
+   let enrichmentItem = {};
```

Comment:
This is not necessary, can be condensed to 
```javascript
async function _enableHTMLEnrichmentItem( item ) {
  return expandObject(
    await TextEditor.enrichHTML( item.system.description.value, {
      async: true,
      secrets: this.actor.isOwner
    } )
  );
}
```

Reply:
ok